index.js

```diff
@@ -80,6 +81,7 @@ module.exports = postcss.plugin(
         validateCustomProperties({
           rule,
           componentName: range.defined,
+          selectorPattern: patterns.componentSelectors,
```

Comment:
`config.selectorPattern` is unused inside of `validateCustomProperties`, isn't it?

Reply:
removed, thanks